### DISCUSSION TITLE
Support multiple appIds for same oauth provider

### DISCUSSION THREAD

**Discussion by jadus** [meteor/meteor#12525] [Feature Requests]
My app is  served on 2 domains (maybe more in the future)
So I have 2 facebook appIds and 2 google appIds in order to have the good appNames in the consent screens.

But accounts-facebook and accounts-google packages seem to be desgined to accept only one appId per provider.

I had to create a copy of each one (accounts-facebook-domain2, accounts-google-domain2) in order to get it work. But it would be great to be able to do this with the original package.

**Comment by StorytellerCZ** [comment]
Maybe updating https://github.com/Meteor-Community-Packages/meteor-partitioner to work might be the solution for this.

**Reply by jadus** [reply]
Thanks, but it won't work for me : data and user accounts are shared in my use case. There's just 2 different "shells" to access it.
